meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx6ullevk/boot.cmd

```diff
@@ -15,4 +15,15 @@ setenv fdt_addr 85800000
 setenv optee_ovl_addr 0x86000000
 setenv fit_addr ${loadaddr}
 
+#Boot firmware updates
```

Comment:
Nit: add space after # (# Boot) to align with boot.cmd from apalis-imx6 (removes one line from a diff).